Tasks/JenkinsQueueJob/task.json

```diff
@@ -99,6 +99,12 @@
             }
         }
     ],
+    "outputVariables" : [
+        {
+            "name" : "JENKINS_JOB_ID",
+            "description" : "The id of the Jenkins Job instance queued by this task. Use this variable in the Jenkins Download task to download the artifacts for this particular job instance."
```

Comment:
ID should be capitalized.  Job should be lowercase.  "Jenkins Download task" should be "Jenkins Download Artifacts task".